src/test/scala/com/databricks/labs/overwatch/utils/StringExtTest.scala

```diff
@@ -1,8 +1,10 @@
 package com.databricks.labs.overwatch.utils
 
 import com.databricks.labs.overwatch.utils.StringExt.StringHelpers
+import org.scalatest.Ignore
```

Comment:
Did you mean to make these changes?

Reply:
yes. There's the import and a commented out //@Ignore. Not sure if these are leftover from the development?

Reply:
@gueniai done